### PR TITLE
D2K - Add Ordos Mission 2b

### PR DESCRIPTION
**Pull request by MustaphaTR**

### CODE REVIEW

File: mods/d2k/maps/ordos-02b/map.yaml [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+		LockColor: True
+		Color: B3EAA5
+		Enemies: Harkonnen, Creeps
+	PlayerReference@Harkonnen:
+		Name: Harkonnen
+		LockFaction: True
+		Faction: harkonnen
+		LockColor: True
+		Color: FE0000
+		Enemies: Ordos, Creeps
```
Comment (ltem) [inline mods/d2k/maps/ordos-02b/map.yaml]: I think we can remove `Creeps` as Enemies for the AI, it has kind of proven itself.

**Comment by ltem** [comment]
:+1: 
Code and gameplay is fine
(There is one tile error but we have to re-import the other maps someday as well so I don't see it as a problem)

**Comment by MustaphaTR** [comment]
I think it would be better if you show the tile error.

**Comment by ltem** [comment]
I'm already on it. It is the tile above the statue to the right of the middle of the map.

**Comment by MustaphaTR** [comment]
Updated.

**Review by abcdefg30 — APPROVED** [review]
After that is fixed. 👍 / ✅

**Comment by MustaphaTR** [comment]
Updated.

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/65b574db63e2f101d53233503ea13fda16782841...305ea7566986989b3242af00a7c8a3aeac4fba0d)

**Pull request merged**